Automattic/jetpack
# Recenter map after closing a marker info window

Fixes https://github.com/Automattic/wp-calypso/issues/74049

## Proposed changes:
This PR recenters the map after closing a marker info window.

### Other information:


## Jetpack product discussion


## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
1. Apply this PR
2. Set the cookie to force the display of Mapkit maps, by pasting this in your console: `document.cookie = "map_provider=mapkit; path=/;";`
3. Add a post/page
4. Add a map block
5. Add some markers
6. Click a marker, this should center the window on this marker
7. Close it by tapping on the map, the old map center should be used.

https://user-images.githubusercontent.com/528287/223455417-b38181e5-9db3-4796-a103-bb01efa09baa.mp4
